src/Riok.Mapperly/Symbols/MethodParameter.cs

```diff
@@ -4,11 +4,24 @@
 
 namespace Riok.Mapperly.Symbols;
 
-public readonly record struct MethodParameter(int Ordinal, string Name, ITypeSymbol Type)
+public readonly record struct MethodParameter
 {
     public MethodParameter(IParameterSymbol symbol)
         : this(symbol.Ordinal, symbol.Name, symbol.Type.UpgradeNullable()) { }
```

Comment:
Couldn't this fix be implement as simple as instead of using `symbol.Name` here using `symbol.ToDisplayString(_parameterNameFormat)` with 
```c#
private static readonly SymbolDisplayFormat _parameterNameFormat = new SymbolDisplayFormat(
        parameterOptions: SymbolDisplayParameterOptions.IncludeName,
        miscellaneousOptions: SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers);
```